src/audio/component.c

```diff
@@ -506,6 +516,46 @@ int comp_copy(struct comp_dev *dev)
 	return ret;
 }
 
+#ifdef SOF_PERFORMANCE_MEASUREMENTS
+void comp_init_performance_data(struct comp_dev *dev)
```

Comment:
does this all new code really belong to component.c?

Reply:
Well... it doesn't need to be here, but those were the methods in "component" class in reference FW.